src/libs/actions/Session.js

```diff
@@ -208,19 +208,20 @@ function restartSignin() {
  *
  * @param {String} password
  * @param {String} validateCode
+ * @param {String} email
  */
-function setPassword(password, validateCode) {
+function setPassword(password, validateCode, email = '') {
```

Comment:
This probably can be non-optional as we are only already defaulting to an empty string when calling this method.

Also NAB since I think we should change this, but optional params are documented with brackets like this...

```js
 * @param {String} [email]
```